### PR TITLE
Add documents

### PR DESCRIPTION
**Pull request by yamatcha**

### CODE REVIEW

File: docs/design.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+#### `autovirtualdc-controller`
+
+A deployment that create and delete VirutualDC resources according to set schedule.
+- startSchedule and stopSchedule are empty: controller create virtualdc immediately
+- startSchedule and stopSchedule are set: calculate NextTime from those schedule and set time to status
+  - nextStartTime < now and nextstopTime < now: do nothing
+  - now < nextStartTime and nextStopTime < now: start virtualdc
+  - nextStopTime < now and now < nextStartTime: stop virtualdc
+  - nextStartTime < now and nextStopTime < now: stop virtualdc
```
Comment (umezawatakeshi) [inline docs/design.md:43]: What happens in case that either `startSchedule` or `stopSchedule` is set?

File: docs/crd_autovirtualdc.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+#### AutoVirtualDCSpec
+
+AutoVirtualDCSpec defines the desired state of AutoVirtualDC
+
+| Field | Description | Scheme | Required |
+| ----- | ----------- | ------ | -------- |
+| template | Template is a template for VirtualDC | VirtualDC | false |
+| startSchedule | StartSchedule is time to start VirtualDC. This format is cron format. | string | false |
+| stopSchedule | StopSchedule is time to stop VirtualDC. this format is cron format. | string | false |
```
Comment (umezawatakeshi) [inline docs/crd_autovirtualdc.md:43]: Should clarify the timezone. UTC or local?

File: README.md [hunk 3]
```diff
@@ -3,10 +3,19 @@
...
+Nyamber
 ============================
+Nyamber is a custom controller to create Neco environment
 
-**Project Status**: Initial development
+## Features
+- Run a dctest in a pod
+- Created a pod which runs dctest and a service to access the pod corresponding to vdc custom resources
+- Run user-defined command
+- Reflect to vdc resources in status of executed command
```
Comment (umezawatakeshi) [inline README.md:14]: Is this English correct...?

suggestion: Reflect status of executed command in the vdc resource.

File: Makefile [hunk 4]
```diff
@@ -83,6 +83,11 @@ test: ## Run tests.
 	$(STATICCHECK) ./...
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) --arch=amd64 use $(ENVTEST_K8S_VERSION) -p path)" go test -v ./... -coverprofile cover.out
 
+.PHONY: apidoc
```
Comment (kmdkuk) [inline Makefile:86]: Please call the apidoc task in check-generate task.

File: docs/design.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+### Kubernetes workloads
+
+#### `virtualdc-controller`
+
+A deployment that controls Runner Pod.
+It create Runner pod which runs dctest and service to access Runner pod.
+
+#### `autovirtualdc-controller`
+
+A deployment that create and delete VirutualDC resources according to set schedules.
```
Comment (kmdkuk) [inline docs/design.md:37]: In addition to schedule, it would be nice to mention retries in case of dctest bootstrap failure.

**Review by umezawatakeshi — APPROVED** [review]

**Review by kmdkuk — APPROVED** [review]
LGTM

**Review by umezawatakeshi — APPROVED** [review]

**Pull request merged**